src/components/Box/Box.tsx

```diff
@@ -160,11 +138,6 @@ export const textStyleVariables = ({
         '--text-size': size ? cssFontSizeToken({ type, size }) : 'inherit',
         '--text-leading': leading ? cssLeadingToken({ type, size: size || 'md', leading }) : 'inherit',
       } as CSSProperties;
-    case 'note':
```

Comment:
Since the only type that can be handled by box is body, we want to be able to use it without specification.
Fixed in another issue.